src/components/ContributeAs.js

```diff
@@ -26,12 +25,10 @@ const SearchIcon = styled(Search)`
 
 const ContributeAsEntryContainer = styled(Container)`
   cursor: pointer;
-  &:hover {
-    background: ${themeGet('colors.black.50')};
-  }
 `;
 
 const messages = defineMessages({
+  anonymous: { id: 'profile.anonymous', defaultMessage: 'anonymous' },
```

Comment:
Can we make it title case to be consistent with the other labels?

```suggestion
  anonymous: { id: 'profile.anonymous', defaultMessage: 'Anonymous' },
```